polynote-kernel/src/main/scala/polynote/kernel/ScalaCompiler.scala

```diff
@@ -600,9 +593,12 @@ object ScalaCompiler {
     global            <- ZIO(new Global(settings, KernelReporter(settings)))
     counter           <- kernelCounter
     notebookPackage    = s"notebook$counter"
-    classLoader       <- makeClassLoader(settings, dependencyClasspath ++ transitiveClasspath).memoize
+    classLoader       <- makeClassLoader(settings, dependencyClasspath ++ transitiveClasspath)
```

Comment:
no longer needs to be memoized? 

(I think we talked at one point about how  `.memoize` didn't do what we thought it did, but it seems at least the intention to memoize was there?)

Reply:
It's no longer passed as an effect... the reasons for having it passed as a `Task[ClassLoader]` are gone for a while, so I changed it to just be a value. So no, it doesn't need to be memoized anymore because it's a simple value

Reply:
Ah, missed that 😄